index.html

```diff
@@ -127,23 +123,24 @@ <h3 class="multi-text-color">Gain+Glory</h3>
 </section>
 
 <!-- About me Session -->
-<section class="about-container">
-    <img class="float-right" src="images/Mask_Group_floate_right.png" alt="elipse-floating-right">  
-      <h2 class="about-me">About Me</h2><br><br><br>
-         <p class="about-message">I can help you build a product, feature or website 
-         Look through some of my work and experience! If you like what you see and have a project you need coded, 
-          don't hestiate to contact me.</p>
-         <div><img class="float-left" src="images/Group_float_left.png" alt="floating-elispe-left">
-  <button class="see-project get-resume" type="button">Get My Resume</button>  
-</div> 
+<section> 
+  <div class="about-container"> 
+    <div class="about-me"><h2>About me</h2></div>      
+        <div class="about-background-desktop-image-2">
+          <p class="about-message">I can help you build a product, feature or website 
+         Look through some of my work and <br> experience! If you like what you see and have a project you need coded,<br>
+          don't hestiate to contact me.</p></div>  
+          <div class="get-resume">
+        <button class="see-project" type="button">Get My Resume</button></div>        
+</div>   
```

Comment:
- [ ] On desktop versions between 768px and 1080, the about section is skewed to the left because it doesn't take up the entire available width. Please fix it 🙏

### Your Design
![image](https://user-images.githubusercontent.com/46839548/166010044-d75840cf-feec-4d46-ae4b-f4339fd5d909.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166010435-0204a8d4-a058-4b0a-ac77-949b8ec5b76a.png)

Reply:
fixed, thanks